codelab-master/apps/playground/src/app/quiz/containers/quiz/quiz.component.scss

```diff
@@ -0,0 +1,124 @@
+@import "variables";
+
+mat-card-header {
+  display: block;
+
+  .header-image {
+    margin: -15px 0 0 15px;
+  }
+
+  mat-card-title {
+    display: inline-block;
+    margin: -100px -130px 0 -120px !important;
```

Comment:
what are those numbers?
Have you considered changing container styles to avoid negative margins?

Reply:
removed, margins for mat-card-title already defined in styles.scss